lib/senkyoshi/canvas_course.rb

```diff
@@ -63,9 +62,49 @@ def self.from_metadata(metadata, blackboard_export = nil)
     # Creates a canvas assignment from a scorm package that has already been
     # uploaded to a scorm manager
     ##
-    def create_scorm_assignment(scorm_package, course_id)
-      url = Senkyoshi.scorm_launch_url +
-        "?course_id=#{scorm_package['package_id']}"
+    def create_scorm_assignment(scorm_package, course_id, local)
+      if local
+        _create_scorm_assignment_local(scorm_package)
+      else
+        _create_scorm_assignment_external(scorm_package, course_id)
+      end
+    end
+
+    ##
+    # Assembles the launch url with the course_id
+    ##
+    def _scorm_launch_url(package_id)
+      "#{Senkyoshi.configuration.scorm_launch_url}?course_id=#{package_id}"
+    end
+
+    ##
+    # Creates a scorm assignment from a Canvas course object
+    ##
+    def _create_scorm_assignment_local(scorm_package)
```

Comment:
Line 67 :] If you look at https://github.com/atomicjolt/senkyoshi_canvas_plugin/pull/1/files#diff-c177a9387bec8249730ca0827fc439c5R151 it calls `process_scorm(local: false)` and that where this is used.